### PR TITLE
feat(frontend): Module consolidation - clean up for OS logic - init virtual assistant

### PR DESCRIPTION
**Pull request by cptran777**
v1:
- Previously, `datahub-web` was made up of too many modules. This PR consolidates some of them into existing modules in order to provide a more organized experience and hopefully also reduce build times. 
- Previous PR for updating datahub-web included some quick fixes for OS logic that was broken. We fixed some of these in a more robust way in the internal code and are pushing some of these changes out now
- Initial implementation for a virtual assistant for DataHub. This is a POC for a future concept available to preview in this code

### Testing Done
```
Manually tested with ./gradlew build and ensured that module tests are passing

Started docker images with latest changes and ensured that application still builds and behaves as expected.
```

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

**Comment by cptran777** [comment]
Note: This is the last (or maybe second to last) clean up / catch up PR for internal stuff. After this we can start tackling some of those new features and issues that have been raised by the community.

File: datahub-web/@datahub/user/LICENSE.md [hunk 1]
```diff
@@ -1,13 +1,9 @@
-Copyright (c) 2020 LinkedIn
+The MIT License (MIT)
```
Comment (keremsahin1) [inline datahub-web/@datahub/user/LICENSE.md:1]: Assuming unintentional change?

**Comment by mars-lan** [comment]
Please follow the [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format) and tag any open issues this might close.

**Comment by igbopie** [comment]
LG

**Pull request merged**